content/docs/command-reference/data/status.md

```diff
@@ -108,6 +117,41 @@ yet, and a file `model.pkl` has been deleted from the workspace. The
 `data/features/` directory is modified, but there is no further details to what
 changed inside. The `--granular` option can provide more information on that.
 
+## Example: Combine with Git status
+
+```dvc
+$ dvc data status
+Not in cache:
+  (use "dvc fetch <file>..." to download files)
+        data/data.xml
+
+DVC committed changes:
+  (git commit the corresponding dvc files to update the repo)
+        modified: data/features/
+
+DVC uncommitted changes:
+  (use "dvc commit <file>..." to track changes)
+  (use "dvc checkout <file>..." to discard changes)
+        deleted: model.pkl
+(there are other changes not tracked by dvc, use "git status" to see)
+
+$ git status
+On branch main
+Your branch is up to date with 'origin/main'.
+
+Changes not staged for commit:
+  (use "git add <file>..." to update what will be committed)
+  (use "git restore <file>..." to discard changes in working directory)
+        modified:   dvc.lock
+
+no changes added to commit (use "git add" and/or "git commit -a")
+```
+
+`dvc data status` and `git status` combined show you the full status of a
+repository. `dvc data status` shows you changes to DVC data, and `git status`
+shows changes to the corresponding `dvc.lock` or `.dvc` files (as well as
+unrelated changes to your Git repository).
```

Comment:
Minor: seems redundant (already explained and hopefully clear from the example title and code block).
```suggestion
```